packages/ui/src/components/Question/Question.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { AccordionButton, AccordionItem, AccordionItemProps, AccordionPanel } from '@chakra-ui/react';
+
+import { OutlinedArrowDownIcon } from '../../icons';
+import { Typography } from '../Typography';
+
+export type QuestionProps = {
+  title: string;
+  content: string;
+} & AccordionItemProps;
+
+export const Question = ({ title, content }: QuestionProps) => {
+  return (
+    <AccordionItem borderColor="gray.200" p="24px 0px">
+      {({ isExpanded }) => (
+        <>
+          <h2>
+            <AccordionButton m="8px 0px" p="0px">
+              <Typography flex="1" textAlign="left" size="lg" weight="medium" color="gray.900">
+                {title}
+              </Typography>
+              <OutlinedArrowDownIcon
```

Comment:
🤔 On our mockups arrow has different color and has no padding on the left
<img width="113" alt="CleanShot 2021-08-06 at 10 42 03@2x" src="https://user-images.githubusercontent.com/22938071/128483321-e1f1002f-3bf9-4509-a612-840a9864ac0e.png">

Reply:
Color is now ok, but there still is a padding that doesn't exist on mockups